application-flashmessages-ui/src/main/resources/Flash/DisplayFlashMessages.xml

```diff
@@ -183,14 +183,20 @@
     }
   #end
     ];
-    window.flashNotifier(params);
+    require(['jquery'], function($) {
+      $(window)[0].flashNotifier(params);
```

Comment:
I'm not sure what is the advantage of using ``$(window)[0]`` over simply ``window``. If there's no difference then I would keep the code simple.